lib/katello/tasks/update_content_default_http_proxy.rake

```diff
@@ -0,0 +1,59 @@
+require File.expand_path("../engine", File.dirname(__FILE__))
+
+namespace :katello do
+  desc "Sets the content default http proxy to an existing http proxy based on supplied URL."
+  task :update_default_http_proxy => :environment do
+    setting = ::Setting::Content.where(name: 'content_default_http_proxy').first
+    options = {}
+    o = OptionParser.new
+    o.banner = "Usage: rake katello:update_content_default_http_proxy [options]"
+    o.on("-n", "--name HTTP_PROXY_NAME") do |name|
+      options[:name] = name
+    end
+    o.on("-u", "--url HTTP_PROXY_URL") do |url|
+      options[:url] = url
+    end
+    o.on("-h, --help", "Prints this help") do
+      puts o
+      exit
+    end
+    ordered_args = o.order!(ARGV) {}
+    o.parse!(ordered_args)
+
+    unless options.key?(:name)
+      $stderr.print("ERROR: Missing required option for --name HTTP_PROXY_NAME")
+      exit 2
+    end
+
+    User.current = User.anonymous_api_admin
+    http_proxy = HttpProxy.where(name: options[:name]).first
+
+    if http_proxy
+      setting.update_attribute(:value, http_proxy.name)
+      puts "Content default http proxy set to #{http_proxy.name_and_url}."
```

Comment:
what if the name is the same, but some option got updated like password?

Reply:
Ok, so if they use the "name" that matches an existing proxy, then this should update that proxy with the new url, in addition to setting the global default proxy?

Reply:
yes, the end goal is that the user can run the installer with the same options they do today, the installer will call this rake task.  Currently the users use it to add or update global proxy configuration.